charts/internal/alicloud-infra/templates/main.tf

```diff
@@ -22,9 +22,16 @@ resource "alicloud_vpc" "vpc" {
 }
 
 resource "alicloud_nat_gateway" "nat_gateway" {
-  vpc_id = {{ required "vpc.id is required" .Values.vpc.id }}
-  specification   = "Small"
-  name   = "{{ required "clusterName is required" .Values.clusterName }}-natgw"
+  vpc_id            = {{ required "vpc.id is required" .Values.vpc.id }}
+  specification     = "Small"
```

Comment:
As we are doing enhancement, shall we enable customer to customized the spec of the NAT Gateway in the PR.

Reply:
I think we should do it in other commit, because it is not very relevant with the purpose of this PR

Reply:
make sense